stl/inc/algorithm

```diff
@@ -6330,6 +6330,22 @@ _CONSTEXPR20 void make_heap(_RanIt _First, _RanIt _Last) { // make [_First, _Las
 #ifdef __cpp_lib_concepts
 namespace ranges {
     // VARIABLE ranges::make_heap
+    // clang-format off
+    template <random_access_iterator _It, class _Pr, class _Pj>
+        requires sortable<_It, _Pr, _Pj>
+    constexpr void _Make_heap_common(_It _First, _It _Last, _Pr _Pred, _Pj _Proj) {
+        // clang-format on
+        // make nontrivial [_First, _Last) into a heap

```

Comment:
What does "nontrivial" mean here?

Reply:
No idea, I think we should fix up the pre-existing occurrences.